src/redefined.resolver.ts

```diff
@@ -27,10 +28,15 @@ export class RedefinedResolver {
             throw Error("“resolvers” option must be a non-empty array or falsy")
         }
 
-        this.resolvers = options?.resolvers || RedefinedResolver.createDefaultResolvers();
+        this.resolvers = options?.resolvers || [];
     }
 
     async resolve(domain: string, networks?: string[], options?: CustomResolverServiceOptions): Promise<ResolverResponse> {
+        if (!this.resolvers.length) {
```

Comment:
Hmmm..
@Rednoll 
Or create `private constructor` and new static async method `create`  instead of this code in `resolve()` ?

Reply:
Extract to separate method and call it at this place (lazy loading). Also add correct handle of fetch error (use default config if fetch will fail)